rlclientlib/utility/context_helper.cc

```diff
@@ -17,6 +17,7 @@ namespace reinforcement_learning { namespace utility {
   const auto multi = "_multi";
   const auto slots = "_slots";
   const auto event_id = "_id";
+  const auto slot_id = "id";

```

Comment:
I think it "_id" is the right choice here given that we are going to generate uuids which are meaningless from learning point of view

Reply:
That is true. So then provided slot ids will not be able to be used as features

Reply:
Oh no actually, this is fine, we can use "id" here since it is in the context and can be used as feature. The uuids are not added in the context

Reply:
we can double check with product team, but I feel it is better to be consistent by default: if we have context when id is used only for joining and does not make sense for learning (when it is auto-generated), it is safer to have this assumption always.

Reply:
switched to `_id`